src/Microsoft.ML.ImageAnalytics/ImageType.cs

```diff
@@ -13,6 +13,12 @@ public sealed class ImageDataViewType : StructuredDataViewType
     {
         public readonly int Height;
         public readonly int Width;
+
+        static ImageDataViewType()
+        {
+            DataViewTypeManager.Register(typeof(Bitmap), new ImageDataViewType());
```

Comment:
Is it OK to not have height and width associated with this `ImageDataViewType`? Should we support setting those through an attribute?

Reply:
The entire image system in ML.NET is not strongly typed. I think to fix this, we need at least one week and it won't be BC.

---
In reply to: [285623486](https://github.com/dotnet/machinelearning/pull/3263#discussion_r285623486) [](ancestors = 285623486)